easybuild/easyconfigs/c/Clang-Python-bindings/Clang-Python-bindings-8.0.0-GCCcore-8.2.0-Python-2.7.15.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'Tarball'
+
+name = 'Clang-Python-bindings'
```

Comment:
@rokzitko This one slipped in accidentally here?

You should sync your `NFFT` branch with current `develop` to clean up this PR...

Don't remove the `Clang-Python-bindings` easyconfig here, otherwise it'll also be removed from `develop`.